### PR TITLE
Feature/dev 8359

### PR DESCRIPTION
**Pull request by Atash3000**

### CODE REVIEW

File: packages/chart/src/hooks/useBarChart.js [hunk 1]
```diff
@@ -274,7 +236,6 @@ export const useBarChart = () => {
     hoveredBar,
     setHoveredBar,
     onMouseOverBar,
-    onMouseLeaveBar,
-    composeSuppressionBars
+    onMouseLeaveBar
```
Comment (adamdoe) [inline packages/chart/src/hooks/useBarChart.js:239]: Can this file be a .ts file?

**Review by adamdoe — CHANGES_REQUESTED** [review]
See feedback and re-open the review when ready.

File: packages/chart/src/components/BarChart/components/BarChart.Vertical.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const BarChartVertical = () => {
   const [barWidth, setBarWidth] = useState(0)
   const [totalBarsInGroup, setTotalBarsInGroup] = useState(0)
   // prettier-ignore
-  const { assignColorsToValues, barBorderWidth, getAdditionalColumn, getHighlightedBarByValue, getHighlightedBarColorByValue, lollipopBarWidth, lollipopShapeSize, onMouseLeaveBar, onMouseOverBar, section, composeSuppressionBars,  fontSize } = useBarChart()
+  const { assignColorsToValues, barBorderWidth, getAdditionalColumn, getHighlightedBarByValue, getHighlightedBarColorByValue, lollipopBarWidth, lollipopShapeSize, onMouseLeaveBar, onMouseOverBar, section,} = useBarChart()
```
[the remark this answers is not in the document]
Comment (Atash3000) [reply packages/chart/src/components/BarChart/components/BarChart.Vertical.tsx:30]: Fixed!

File: packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx [hunk 3]
```diff
@@ -258,7 +219,7 @@ export const BarChartHorizontal = () => {
                             verticalAnchor='middle'
                             fontWeight={'normal'}
                           >
-                            {barLabel}
+                            {defaultBarLabel}
```
Comment (adamdoe) [inline packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx:222]: I'm not seeing a definition for defaultBarLabel

**Review by adamdoe — CHANGES_REQUESTED** [review]

File: packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx (hunk 3 above)
Comment (Atash3000) [reply packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx]: Fixed!

**Pull request merged**